### PR TITLE
CRYO VISION OH SHIT BOI

### PR DESCRIPTION
**Pull request by Geevies**
* Examining a cryopod allows you to see who's in it, if there's someone in it.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/machinery/cryopod.dm [hunk 1]
```diff
@@ -217,6 +217,11 @@
 	icon_state = base_icon_state
 	find_control_computer()
 
+/obj/machinery/cryopod/examine(mob/user)
+	..(user)
+	if(occupant)
+		to_chat(user, "[occupant] is inside \the [src].")
```
Comment (Sindorman) [inline code/game/machinery/cryopod.dm]: I think here `is` should be generic or depend on the occupant. It could be `are`

Comment (OneOneThreeEight) [reply code/game/machinery/cryopod.dm]: That may lead to grammar issues.

`John Doe are inside the cryopod.`

Comment (Geevies) [reply code/game/machinery/cryopod.dm]: Yeah, I think `is` is the best one to use.

Comment (MarinaGryphon) [reply code/game/machinery/cryopod.dm]: `The bees is inside the cryopod.`
:thinking: 
Just use `gender_datums[occupant.gender].is`.

Comment (Geevies) [inline code/game/machinery/cryopod.dm]:
```suggestion
		to_chat(user, "[occupant] [gender_datums[occupant.gender].is] inside \the [src].")
```

**Pull request merged**